docs/src/main/asciidoc/ide-tooling.adoc

```diff
@@ -22,7 +22,7 @@ In addition, IntelliJ IDEA has additional support for Quarkus in their Ultimate
 
 The table below gives an overview of the current IDEs with links and a high-level overview of their features.
 
-:vscode-logo: https://simpleicons.org/icons/visualstudiocode.svg
+:vscode-logo: {imagesdir}/visual-studio-code.svg
 :eclipse-logo: https://simpleicons.org/icons/eclipseide.svg
 :intellij-logo: https://simpleicons.org/icons/intellijidea.svg
 :che-logo: https://simpleicons.org/icons/eclipseche.svg
```

Comment:
Maybe we should have all the logos hosted in our website instead of linking external resources?

Could you take care of this?